plot-lib/plot/private/plot2d/plot-area.rkt

```diff
@@ -375,7 +375,7 @@
 
     (: get-x-far-tick-label-params (-> (Listof Label-Params)))
     (define (get-x-far-tick-label-params)
-      (if (and (plot-x-far-axis?) draw-x-far-tick-labels?)
+      (if (or (and (plot-x-far-axis?) draw-x-far-tick-labels?) (and (plot-x-far-axis?) (plot-x-far-tick-labels?)))
```

Comment:
I think that `draw-x-far-tick-labels?` should be updated to include `plot-x-far-tick-labels?` in the test and this line should remain unchanged.  There are other methods which might be affected by the position of the plot labels, for example `max-x-far-tick-label-height` and `get-x-far-tick-label-params`.